gcloud-workstation-phpstorm/scripts/011_install-mdless.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-sudo -u user "gem install --user-install mdless" || true
+sudo -u user gem install --user-install mdless || true
```

Comment:
was assuming we will remove this from the base install when introducing the custom scripts.

Reply:
removed now